### ISSUE TITLE
.rpgmvp extraction failed

### ISSUE THREAD

**Issue by AestheticPerfection** [Bioruebe/UniExtract2#114]
Hiya. I just noticed this morning, trying to extract .rpgmvp files gives me an error - and even googling for said file didn't help me in the slightest. Any ideas?
[2018-10-01_07-11-06_FAILED_イベント2薬師1.rpgmvp.log](https://github.com/Bioruebe/UniExtract2/files/2432652/2018-10-01_07-11-06_FAILED_.2.1.rpgmvp.log)
![f6ey 1](https://user-images.githubusercontent.com/38237339/46270749-7fab0900-c549-11e8-9391-4a903869d375.png)

**Comment by TheGuyInShades** [comment]
Well, from looking at the log it looks like some of the folder and file names are in Japanese, that might be part of the issue.  I don't know what gcmt-dll.dll is for but I found it, maybe putting it in the UniExtract\bin folder might help. https://www.dropbox.com/s/jl7ckgjups8qfhb/gcmt-dll.zip?dl=0

**Comment by AestheticPerfection** [comment]
After copying in the gcmt-dll.dll and trying a file without japanese characters I get a new error. 
I tried googling "rmvdec" to find update or other libraries, but I couldn't find anything of that sort. 
[2018-10-02_07-43-58_FAILED_001_01.rpgmvp.log](https://github.com/Bioruebe/UniExtract2/files/2436591/2018-10-02_07-43-58_FAILED_001_01.rpgmvp.log)
![3qva 1](https://user-images.githubusercontent.com/38237339/46331251-4214b180-c617-11e8-9a38-7d22240d93f8.png)

**Comment by TheGuyInShades** [comment]
Yeah, I've been testing it with an rpgmvp file I have, give me a little bit to try to figure out the problem.

**Comment by TheGuyInShades** [comment]
First of all, you can delete that gcmt-dll.dll file I gave you, I don't think it works right. Just rename the gc.dll file that should already be in UniExtract\bin to gcmt-dll.dll, maybe backup the gc.dll file first just in case.
I've been having a completely different issue with , I can't seem to figure out what the problem is. rmvdec.exe seems to have been made with a program called Haxe, that's where the file gcmt-dll.dll came from anyway. https://haxe.org  Maybe rmvdec.exe wasn't compiled correctly or something? I wish I could be more help. 
[2018-10-01_23-22-27_FAILED_112.rpgmvp.log](https://github.com/Bioruebe/UniExtract2/files/2436702/2018-10-01_23-22-27_FAILED_112.rpgmvp.log)

**Comment by AestheticPerfection** [comment]
After making a copy of the gc.dll and renaming said copy to gcmt-dll.dll and leaving the gc.dll as it is, it works just fine. Also, I just checked the website and it seems the gcmt-dll.dll is needed for the Neko compiler stuff in Haxe? Not sure about that. Though, the /bin folder has a neko.dll in it.

**Comment by TheGuyInShades** [comment]
Oh, you got it working? Cool! I copied the gc.dll file first too, but it doesn't seem to make a difference having both gc.dll and gcmt-dll.dll so I just kept gcmt-dll.dll. I'm not really concerned about getting it working for myself, as long as you got it working then it's cool.
